### PR TITLE
Add example for commit status on group steps

### PR DESCRIPTION
**Pull request by pzeballos**

### CODE REVIEW

File: pages/integrations/github.md [hunk 1]
```diff
@@ -142,6 +142,30 @@ For example, if you have a monorepo containing three applications, you could use
...
+        command: ".buildkite/steps/brakeman"
+      - label: "\:bundleaudit\: Bundle Audit"
+        command: ".buildkite/steps/bundleaudit"
+      - label: "\:yarn\: Yarn Audit"
+        command: ".buildkite/steps/yarn"
+      - label: "\:yarn\: Outdated Check"
+        command: ".buildkite/steps/outdated"
+```
+
+In this case, you'll have a "logical AND"-style behaviour for the notification.
```
Comment (mbelton-buildkite) [inline pages/integrations/github.md:167]: Can you please unpack this a bit for for me? Does the logical AND relate to the notification only being sent when all pass/complete, or that it's applied to all steps, or something else?

Comment (pzeballos) [reply pages/integrations/github.md:167]: Hey! not sure I understand your question. The logical AND is for the commit status of each step from the group

Comment (mbelton-buildkite) [reply pages/integrations/github.md:167]: All good. It's not clear to me what the logical AND behaviour relates to. Can you tell me what the inputs to the logical AND are, and what happens when the result is true vs. false?

(My guesses:

- It means a notification is only sent when all steps reach some status (passed? completed?).
- It means the custom label must be applied to all checks listed in the steps, not just a few. I'm not sure if they display individually or a group.)

Comment (pzeballos) [reply pages/integrations/github.md:167]: Notifications are always sent. What this means is that if you have a group of steps where the first job returns Passed and the second Failed, the commit status that will appear in GitHub is Failed (and only one commit status will appear)

Comment (mbelton-buildkite) [reply pages/integrations/github.md:167]: Go it! Thanks :)

How about:

```suggestion
GitHub displays one commit status for a group, meaning a pass result only displays when all jobs in the group pass.
```

What do you think?

Comment (pzeballos) [reply pages/integrations/github.md:167]: sounds good!; what it's a bit confusing for me though, is that GitHub will display only one commit status if the `github_commit_status` is set at a group level, but it's possible to set `github_commit_status` on each job of the group and those will be treated individually :)

Comment (mbelton-buildkite) [reply pages/integrations/github.md:167]: Ahhhh okay. How about:

```suggestion
When you set a custom commit status on a group step, GitHub only displays one status for the group. A passing result only shows when all jobs in the group pass. If you want to show custom commit statuses for each job, set them on the individual step.
```

Comment (pzeballos) [reply pages/integrations/github.md:167]: Much clearer! 🙌

**Review by mbelton-buildkite — APPROVED** [review]
Nice team work! Looks good to me :)

**Comment by mbelton-buildkite** [comment]
@pzeballos, you should be able to resolve the failing build by merging the latest master. There were some broken links that are now fixed :)

**Comment by mbelton-buildkite** [comment]
@pzeballos, I'm just skimming through old PRs to see if I can resolve any quickly and this looks like a good candidate. I'll update the branch and merge. Let me know if there are any issues and we can revert. 😊

**Comment by pzeballos** [comment]
Thanks @mbelton-buildkite! I completely missed this one!

**Pull request merged**